### PR TITLE
Add dlp v2beta1

### PR DESCRIPTION
**Pull request by eoogbe**

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -0,0 +1,1875 @@
+{
```
Comment (landrito) [inline package-lock.json]: Was this file added by mistake?

Comment (eoogbe) [inline package-lock.json]: It was generated when I ran `npm install` and it said to commit it.

Comment (landrito) [inline package-lock.json]: Hm. I'm a bit unfamiliar with how this file works. @stephenplusplus is this file correct?

Comment (stephenplusplus) [inline package-lock.json]: I'm indifferent if we want to include it or not. It does not get published, or affect users (afaik), but is for our benefit as the developers of this library. More info on this here: https://docs.npmjs.com/files/package-lock.json - let me know if you catch anything I didn't.

Comment (eoogbe) [inline package-lock.json]: Looks pretty similar to a `Gemfile.lock` or a `yarn.lock`. We ignore `Gemfile.lock`, should we ignore this file as well?

Comment (landrito) [inline package-lock.json]: Lets go ahead and .gitignore it.

**Comment by landrito** [comment]
LGTM after .gitignore.

**Pull request merged**